script.js

```diff
@@ -0,0 +1,19 @@
+
+var arr = new Array();
+function myFunction(name,breed,fee){
+  alert("Name:"+name+"\nBreed:"+breed+"\nAdoption Fee:"+fee);
+}
+
+function getSum(total,num) {
+  return total + Math.round(num);
+}
+
+function adoptFunction(item){
+  var x = document.getElementById("price").childNodes[0].nodeValue;
```

Comment:
This always targets the first price element on your page and is a little redundant.